### DISCUSSION TITLE
Where to find devtool sourcemap configurations

### DISCUSSION THREAD

**Discussion by simPod** [webpack/webpack#15457] [Q&A]
[Docs says](https://webpack.js.org/configuration/devtool/):

> The devtool option adds the plugin internally so you would end up with the plugin applied twice.

Can I somewhere see what does each option of `devtool` maps to? I did not find anywhere in the sourcecode. Was looking for `inline-source-map` to see what plugin config/other options it sets.

**Comment by vankop** [comment]
non eval source maps https://github.com/webpack/webpack/blob/main/lib/SourceMapDevToolPlugin.js
eval source maps https://github.com/webpack/webpack/blob/main/lib/EvalSourceMapDevToolPlugin.js

**Reply by simPod** [reply]
Thanks for links. I still kind of struggle to find what e.g. `devtool: inline-source-map` produces. I quickly went over the code + cmd+f inline.

**Reply by vankop** [reply]
inline just inline "default" source maps in comment, more info in docs https://webpack.js.org/configuration/devtool/

**Reply by simPod** [reply]
So when I disable `devtool`, how do I configure `SourceMapDevToolPlugin` to behave the same, that said inline source maps in comment? I wanted to check the source so I didn't have to ask.
